mobile/Ihc/services/LocalDataService.js

```diff
@@ -310,6 +329,38 @@ export function markPatientsUploaded() {
    }
  */
 
+export function handleDownloadedMedications(medications) {
```

Comment:
Can you move the comments from line 324 right above handleDownloadedPatients? It is referring to that function